app/src/main/res/raw/munge_email.js

```diff
@@ -76,11 +76,6 @@ function normalizeElementWidths(elements, webViewWidth, messageUid) {
             logInfo(`Zoom updated: documentWidth / element.scrollWidth -> ${documentWidth} / ${element.scrollWidth} = ${newZoom}.`);
             element.style.zoom = newZoom;
         }
-
-        if (document.documentElement.scrollWidth > document.documentElement.clientWidth) {
-            logInfo(`After zooming the mail it can still scroll: found clientWidth / scrollWidth -> ${document.documentElement.clientWidth} / ${document.documentElement.scrollWidth}`);
-            reportOverScroll(document.documentElement.clientWidth, document.documentElement.scrollWidth, messageUid);
-        }
```

Comment:
We could just keep the javascript logs to the console and simply not report the overscroll instead